install/kubernetes/helm/istio/values.yaml

```diff
@@ -391,6 +391,7 @@ grafana:
   enabled: false
   replicaCount: 1
   image: grafana
+  persist: false
```

Comment:
missing grafana.storageClassName in values.yaml.

Reply:
I used the [`default`](https://github.com/helm/helm/blob/master/docs/chart_template_guide/functions_and_pipelines.md#using-the-default-function) function in helm to do this in chart since its not required. See here: https://github.com/istio/istio/pull/7553/files#diff-ff7108680e9d83df61f2cd529dea8375R12

I can also include it here if it's clearer.

Reply:
right - the reason to document in values.hyaml is so that we can autogenerate documentation from values.yaml.